x/marker/keeper/msg_server.go

```diff
@@ -752,3 +752,63 @@ func (k msgServer) SetAccountData(goCtx context.Context, msg *types.MsgSetAccoun
 
 	return &types.MsgSetAccountDataResponse{}, nil
 }
+
+// UpdateSendDenyList updates the deny send list for restricted marker. Signer must be admin or gov proposal.
+func (k msgServer) UpdateSendDenyList(goCtx context.Context, msg *types.MsgUpdateSendDenyListRequest) (*types.MsgUpdateSendDenyListResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	marker, err := k.GetMarkerByDenom(ctx, msg.Denom)
+	if err != nil {
+		return nil, fmt.Errorf("marker not found for %s: %w", msg.Denom, err)
+	}
+
+	if marker.GetMarkerType() != types.MarkerType_RestrictedCoin {
+		return nil, fmt.Errorf("marker %s is not a restricted marker", msg.Denom)
+	}
+
+	if msg.Authority == k.GetAuthority() {
+		if !marker.HasGovernanceEnabled() {
+			return nil, fmt.Errorf("%s marker does not allow governance control", msg.Denom)
+		}
+	} else {
+		if !marker.HasAccess(msg.Authority, types.Access_Transfer) {
```

Comment:
I feel like `Transfer` access isn't the correct thing to use here. We probably want a separation of concerns between "people who can transfer the coin" and "people that control who can transfer the coin."

The `Admin` access might actually be correct here. It was originally for "People that can add/remove access", which is fairly in line with what's being done here.

Alternatively, we might need a new access type, and that name might be hard to iron out. My initial though is `manager`, but I worry that that will get confused with the marker field with that same name. I feel like that's what we're still trying to convey, though. There's a few endpoints now that might make more sense using such a permission.

Thoughts?

Reply:
`transfer` literally is intended as "transfer authority" ... so they can delegate/pre-approve via kyc, etc if they wish but ultimately they are responsible for controlling the transfer of the token asset.  "People that are approved to transfer" are only approved if the `transfer` holding authority has approved it.

Reply:
It makes sense that it would be `transfer` instead of `admin` or something else.  Transfer authority also controls the force transfer.  Therefore, a transfer authority account could put an address on a send deny list then force transfer those markers to another address.   This would fill a requirement of recovery of a coin if the holder has lost access to their private keys.

Reply:
I believe they would want to use `authz` to authorize `MsgTransferRequest` which isn't going to let someone actually perform any of the operations that explicitly check for `transfer` grants held by and address...  I am not aware of anyone actually giving someone the `transfer` grant directly just so they can move the tokens.